sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -579,36 +581,66 @@ static inline int find_codec_info_acpi(const u8 *acpi_id)
 	return i;
 }
 
-static int get_sdw_dai_link_num(struct snd_soc_acpi_mach_params *mach_params)
+/*
+ * get BE DAI and CPU DAI number based on sdw link adr.
+ * Since some sdw links may be aggregated to support one
+ * PCM, CPU DAI number is not always equal to BE DAI.
```

Comment:
"CPU DAI Number is not always equal to BE DAI" seems incomplete, not always equal to BE DAI what?

Reply:
@RanderWang I suggested some changes for the comments from @ranj063 in my diff.

Reply:
thanks, I will check it